playbooks/roles/openvpn/tasks/main.yml

```diff
@@ -139,7 +158,11 @@
   template: src=etc_openvpn_server.conf.j2
             dest=/etc/openvpn/server.conf
 
-- name: Restart OpenVPN so the 10.8.0.0 interface is available to dnsmasq
+- name: Copy OpenVPN UDP configuration file into place
+  template: src=etc_openvpn_server_udp.conf.j2
+            dest=/etc/openvpn/server-udp.conf
+
+- name: Restart OpenVPN so the 10.8.0.0/10.9.0.0 interface is available to dnsmasq
```

Comment:
Let's rephrase this as "Restart OpenVPN so the TUN interfaces are available to dnsmasq".